src/data/CassandraBackend.hpp

```diff
@@ -849,6 +894,20 @@ class BasicCassandraBackend : public BackendInterface {
         executor_.write(std::move(statements));
     }
 
+    void
+    writeMPTHolders(std::vector<std::pair<ripple::uint192, ripple::AccountID>>&& data) override
```

Comment:
Do we really need r value here? seems no difference if we just pass a const&

Reply:
Will change